cogs5e/sheets/beyond.py

```diff
@@ -359,9 +362,15 @@ def get_ac(self):
             if val['typeId'] == 1:  # AC override
                 return val['value']
             elif val['typeId'] == 2:  # AC magic bonus
-                miscBonus += val['value']
+                try:
```

Comment:
Rather than duplicating a try/except block here, it seems like it would be more elegant to just add `if val['value'] is None: continue` at the start

Reply:
Hadn't thought of that, good point.

Reply:
Newest commit uses `if value None: continue` rather than Try/Except